cdap-docs/admin-manual/source/installation/cloudera.rst

```diff
@@ -122,6 +122,10 @@ available services which CM can install.
 +------------+-----------------------+--------------------------------------+
 | CM Version | CDH Version           | CDAP Parcel / CSD Version            |
 +============+=======================+======================================+
+| 5.9        | 5.9.x                 | |short-version-x|                    |
```

Comment:
CDAP 3.5  (technically 3.5.2, 3.6.1) and up support CDH 5.8 and 5.9, per https://issues.cask.co/browse/CDAP-6234 and https://issues.cask.co/browse/CDAP-7291